Kiosk/App/Networking/Networking.swift

```diff
@@ -34,7 +47,9 @@ class OnlineProvider<Target> where Target: Moya.TargetType {
 
 protocol NetworkingType {
     associatedtype T: TargetType, ArtsyAPIType
-    var provider: OnlineProvider<T> { get }
+    var provider: OnlineProvider<T> {
```

Comment:
That one would be nicer on a single line I guess.

Reply:
Yeah, I agree on single-lining these